test/e2e/data/infrastructure-metal3/main/metadata.yaml

```diff
@@ -1,6 +1,9 @@
 apiVersion: clusterctl.cluster.x-k8s.io/v1alpha3
 kind: Metadata
 releaseSeries:
+- major: 1
```

Comment:
We need 1.8 branch on CAPM3 before adding an infra contract. Let's add this once we have a new branch.